

func main() -> int
{
    var p = 0;
    
    print(print(print(10 / p)));
    print(20 / p);

    assert(false);

    0
}
catch (division_by_zero)
{
    prints("caught division_by_zero\n");
    -111
}



